battle_ship_runner.rb

```diff
@@ -111,15 +111,33 @@ def computer_place_ships(ship)
 #computer takes a shot
 #updates on player board
 #renders player board
-until @player_board.cruiser.sunk = true && @player_board.submarine.sunk = true ||
-  @computer_board.cruiser.sunk = true && @computer_board.submarine.sunk = true
-  fired_upon_cell = @player_board.cells.to_a.sample
-  puts "My turn. I fire upon #{fired_upon_cell}"
-  puts "choose a cell to fire upon:\n"
-  print @computer_board.render(true)
-
-  input = gets.chomp
-
-
+def turn
```

Comment:
awesome start on this method! we can look at the first until loop and find a way to take multiple turns until all ships are sunk on one or the other board.